src/modules/retrocompat-config.js

```diff
@@ -33,12 +31,11 @@ function envVars(config) {
   }
 
   if (process.env.AMQP_DEBUG) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)